### ISSUE TITLE
Home Assistant Color for status (sleepling - mowing)

### ISSUE THREAD

**Issue by ottelo9** [geertmeersman/robonect#190]
Hi there. Thanks for your wonderful HACS integration!

I have few problems and questions regarding the integration.
I have a "Gardena Sileno Smart" = "R100LiC" with robonect (V1.3b) and Home Assistant (up2date) with your actual HACS integration. I activated MQTT and Rest.

- Lot of my entities in HA are always Unknown and I cannot use it. E.g. sensor.mower_mower_error or mower_status, substatus.
- Status of the mower is showing correct (vacuum.mower_robonect) but always in the same color blue. So there is no difference between mowing and sleeping. Can you change it?
- 
Thank you for your great work!

**Comment by geertmeersman** [comment]
hi Otello.
Could you enable debug  (as described in the readme), reload the integration, and stop debug and send me the logs?
You can always do this through discord in a private message if preferred.

**Comment by ottelo9** [comment]
@geertmeersman 
Sorry for the late answer and thanks for your help. You can download the log file from [here](https://drive.google.com/file/d/1KkNH1sYc20d1_xkJLr77b5VZrRdW_Ph3/view?usp=sharing).

I will then delete the file as soon as you answer me. And again, thanks for your help :)

**Comment by geertmeersman** [comment]
Now it's ok 👍

**Comment by geertmeersman** [comment]
Sorry for the delay, only had time now to check. I have the impression that the sensors you refer to, are not being pushed on the MQTT broker.
Could you provide me a view on the topics you have on your mqtt broker for the mower. And is your MQTT topic correctly configured in the integration?

**Comment by ottelo9** [comment]
I use Mosquitto Broker in HA since years for my other devices. My mower (the topic) is correctly configured inside the integration (is there a way to view the configured topic in the integration? I cant find it inside the configuration). 

Here are all available topics from the mower (mqtt explorer):
![2024_08_25_13_17_14_Home_Assistant](https://github.com/user-attachments/assets/ca8f9352-71ef-4811-b3f8-f05c7358744d)

Robonect MQTT (German):
![2024-08-25 13_24_04-MQTT-Client - Sileno Smart](https://github.com/user-attachments/assets/0271eab0-feda-4edb-bd58-b13e1d063ac8)

**Comment by geertmeersman** [comment]
Hi, sorry to only come back on it now, have been quite  busy.
It's certainly not linked to the language setting.
The configured topic is mower, so that seems  fine. Could you rerun the debug  instructions and send me back the log file. Perhaps I'll add some  extra logging to see where it's going wrong

**Comment by ottelo9** [comment]
Yes no problem. Do I have to wait for a new update before logging?

**Comment by ottelo9** [comment]
You can download the log file from [here](https://drive.google.com/file/d/1a-lDzEwhfg-XINDN2p9h3GydxXszJWAA/view?usp=sharing).

**Comment by geertmeersman** [comment]
Got it

**Comment by geertmeersman** [comment]
Can you give me an extraction from home assistant on the sensors of your integration? So I can see which mqtt it registers  correctly and which it doesn't

**Comment by ottelo9** [comment]
I made few screenshots of the entity robonect list. All other ways to extract a list seems to be complicated... I hope its ok for you?

![robonect entities](https://github.com/user-attachments/assets/106b6d73-5edd-4a82-aa58-e9a1a874bda5)

**Comment by geertmeersman** [comment]
Thx, but I would need to see their status too.
Btw if you can join on discord, this can also speed up things

**Comment by ottelo9** [comment]
Do you know a good way to extract these infos?

**Comment by geertmeersman** [comment]
You can go to the developer  tools in HA / States  and then capture the screen in chrome with this extension: https://chromewebstore.google.com/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?pli=1

**Comment by ottelo9** [comment]
Wow, a very, very useful extension, thanks for the tip :)

Here is the captured image:
https://drive.google.com/file/d/1vpCC9EcmKa8M4-et4BgOoF0xhmA8T8Rw/view?usp=sharing

**Comment by geertmeersman** [comment]
Could your remove the integration,  restart HA and add it again? Just to be sure to not have "old" entities

**Comment by ottelo9** [comment]
I removed the Integration, restarted HA and addded it again. I choose these sensors:
![image](https://github.com/user-attachments/assets/08b0151e-c56e-4d81-836f-6c65a9de9147)


mower states as image:
https://drive.google.com/file/d/1rq3D_BVte9-KzBghqPdwed9y32rFc1lP/view?usp=sharing
as pdf:
https://drive.google.com/file/d/1d0w5cHSqTvDP9BbM2k20OBwt9xhar8d3/view?usp=sharing

**Comment by geertmeersman** [comment]
All your entities are fine now, nothing to worry about. That the timers are displayed  is normal, as they are  pushed  through MQTT. You can see that in the attributes, if it comes  from REST or MQTT.
The sensory types of choice, which you refer to in your screenshot, only goes  for the REST information. I could set up a  condition on it, but not a real  priority I think.

**Comment by ottelo9** [comment]
Ok, but the current error is still "unknown". And the color for the vacuum.mower_robonect is always this green, doesnt matter if sleeping or parking ...

Whats about the default disabled entities? I can see them via mqtt explorer (see screenshot at my post at 25.08.2024). If I enable them, they are all unknown.
![image](https://github.com/user-attachments/assets/ef0ff076-6d43-43b9-bae4-2a1c79af9064)

**Comment by ottelo9** [comment]
Here is a small overview of the missing entities that I mean:
![2024_09_26_08_00_07_Home_Assistant](https://github.com/user-attachments/assets/e1dab5ee-ab71-4707-a930-d97bbba05697)

**Comment by geertmeersman** [comment]
Normally they get their value when their value changes on the broker, but perhaps if you enable them and restart HA, it might help?

**Comment by ottelo9** [comment]
After a restart, these entities are now Unknown:
![image](https://github.com/user-attachments/assets/92a47f3c-bce5-4252-80e4-14629b9f05a5)
